other/materials_designer/specific_examples/heterostructure_silicon_silicon_dioxide_hafnium_dioxide_titanium_nitride.ipynb

```diff
@@ -7,13 +7,20 @@
     "collapsed": false
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.        MaterialWithBuildMetadata.create(Materials.get_by_name_first_match(STRUCTURE_PARAMS[0].name)),
</pre><p>Lets just do this for all the <code>STRUCTURE_PARAMS</code> members, e.g. <code>crystals =  [MaterialWithBuildMetadata.create(Materials.get_by_name_first_match(parameter.name)) for parameter in STRUCTURE_PARAMETERS]</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/238/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>